cybered/cybered/settings.py

```diff
@@ -0,0 +1,100 @@
+"""
+Django settings for cybered project.
+
+Generated by 'django-admin startproject' using Django 2.1.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Are we guaranteed that __file__ will always be in the root of the project?

Reply:
`__file__` refers to the current file (except for certain circumstances where it's unset) so as long as `settings.py` doesn't move we're good.